sklearn/model_selection/_validation.py

```diff
@@ -2327,7 +2336,22 @@ def validation_curve(
     fit_params : dict, default=None
         Parameters to pass to the fit method of the estimator.
 
-        .. versionadded:: 0.24
+        .. deprecated:: 1.6
+            This parameter is deprecated and will be removed in version 1.8. Use
+            ``params`` instead.
+
+    params : dict, default=None
+        Parameters to pass to the `fit` method of the estimator and to the scorer.
+
+            - If `enable_metadata_routing=False` (default):
+              Parameters directly passed to the `fit` method of the estimator.
+
+            - If `enable_metadata_routing=True`:
+              Parameters safely routed to the `fit` method of the estimator.
+              See :ref:`Metadata Routing User Guide <metadata_routing>` for more
+              details.
```

Comment:
Same comment here. Since it's passed to more than the estimator `fit`. There's the scorer and CV

Reply:
Thank you @adam2392, that comment also made it more clear for myself.